lib/cell.rb

```diff
@@ -0,0 +1,24 @@
+
+class Cell
+  attr_accessor :mine, :hidden, :flagged, :adjacent_mines
+  def initialize(mine=false, hidden=true, flagged=false, adjacent_mines=0)
+    @mine = mine
+    @hidden = hidden
+    @flagged = flagged
+    @adjacent_mines = adjacent_mines
+  end
+
```

Comment:
I know we talked about this in the scrum on Saturday but I'll say it again... I think the Cell class should have more under the hood so you're not directly changing variables from outside the class. i.e. `cell.place_mine` instead of `cell.mine = true` by way of `attr_accessor`